pages/dynamic-portal.tsx

```diff
@@ -193,6 +194,8 @@ const DynamicTemplates: NextPageWithLayout<Props> = ({
   const spaceProps = {
     accessToken,
     environmentId,
+    name: "Dynamic-templates",
```

Comment:
If this name is visible to the user do `Dynamic Templates`

Reply:
@johnmosesman - Should I put Dynamic Portal instead to match the sidebar link name?

Reply:
Setting it now as Dynamic Templates. I'll change it if we need